Repositories/Repository.cs

```diff
@@ -20,283 +20,608 @@ public class Repository : IRepository {
     }
     
     // Get Queries -----------------------
+    
```

Comment:
Add documentation comments to methods (type /// and a template will be generated for you)